rskj-core/src/test/java/co/rsk/peg/BridgeSupportReleaseBtcTest.java

```diff
@@ -98,6 +99,8 @@ class BridgeSupportReleaseBtcTest {
     private BridgeSupportBuilder bridgeSupportBuilder;
     private SignatureCache signatureCache;
     private FeePerKbSupport feePerKbSupport;
+    private FederationSupportBuilder federationSupportBuilder;
+    private FederationSupport federationSupport;
```

Comment:
I'd remove this one, each test uses it's own

Reply:
done